Source/Lib/Codec/EbFullLoop.c

```diff
@@ -525,6 +526,7 @@ void UnifiedQuantizeInvQuantize_R(
 			temporalLayerIndex < 3 ? EB_TRUE : EB_FALSE,
             (EB_U8)0,
             (EB_U16)qp,
+            bitDepth,
```

Comment:
Do I have to set this to 8bit?

Reply:
Yes, it has been set to 8bit as follows.
https://github.com/OpenVisualCloud/SVT-HEVC/blob/882f926e02711d1825c62202e578c1d6e6839f43/Source/Lib/Codec/EbFullLoop.c#L488

Reply:
thanks for catching this, it seems a pretty bad one for 10bit.